benchmarking/smartseq2/RunBenchmarkingAnalysis.wdl

```diff
@@ -0,0 +1,55 @@
+import "BenchmarkingTasks.wdl" as analysis
+
+workflow RunBenchmarkingAnalysis {
+  String base_datafile
+  String updated_datafile
+  String output_name
+  String gtf_file
+  String metadata_file
+  String metadata_keys
+  String metKeys
+  String base_metrics
+  String updated_metrics
+  Int    npcs
+
+  meta {
```

Comment:
meta should be above the variable declarations on line 4 and have an empty line underneath